code/datums/ruins/lavaland.dm

```diff
@@ -142,3 +142,9 @@
 	id = "codelab"
 	description = "A Nanotrasen genetic research facility, abandoned and ripe for looting. Whats that goo over there?"
 	suffix = "lavaland_surface_codelab.dmm"
+/datum/map_template/ruin/lavaland/codelab
+	name = "Train station"
+	id = "train_station"
+	description = "A Old and overun train station who built it and what were they moving?"
```

Comment:
```suggestion
	description = "An old and abandoned train station. Who built it, and what were they moving?"
```

Reply:
is that all that is requested?

Reply:
you also have to address the discord changes the other maptainer gave you.

Reply:
> you also have to address the discord changes the other maptainer gave you.

I do believe that with the new commit ive addressed those changes now